tests/foreman/ui/test_errata.py

```diff
@@ -1663,7 +1665,9 @@ def test_content_host_errata_search_commands(
     cvs = [module_cv.read(), content_view.read()]  # client0, client1
     # walrus-0.71-1.noarch (RHSA), kangaroo-0.1-1.noarch (RHBA)
     packages = [FAKE_1_CUSTOM_PACKAGE, FAKE_4_CUSTOM_PACKAGE]
-    with Broker(nick='rhel8', host_class=ContentHost, _count=2) as clients:
+    with Broker(
+        nick='rhel10', workflow='deploy-template', host_class=ContentHost, _count=2
+    ) as clients:
```

Comment:
@damoore044 If we already have the nick defined in broker_settings, we might not require workflow arg here

Reply:
will circle back to this and test locally